tests/tests.js

```diff
@@ -1396,4 +1431,43 @@ function multiPeriodTests(manifestUrl, format) {
     // Be more tolerant with float comparison, (D > 1.9 * length) instead of (D == 2 * length).
     expect(video.duration).toBeGreaterThan(1.9);
   });
+}
+
+function lowLatencyDashTests(manifestUrl, format) {
+  it('can process LL-DASH streaming ' + format, async() => {
+    const inputConfigDict = {
+      'inputs': [
+        {
+          'name': TEST_DIR + 'BigBuckBunny.1080p.mp4',
+          'media_type': 'video',
+          // Keep this test short by only encoding 4s of content.
+          'end_time': '0:04',
+        }
+      ],
+    };
+    const pipelineConfigDict = {
+      'streaming_mode': 'live',
+      'resolutions': ['144p'],
+      'video_codecs': ['h264'],
+      'manifest_format': ['dash'],
+      'segment_size': 2,
+      'is_low_latency_dash': true,
+      'utc_timings': [
+        {
+          'scheme_id_uri':'urn:mpeg:dash:utc:http-xsdate:2014',
+          'value':'https://time.akamai.com/?.iso'
+        },
+      ],
+    };
+    await startStreamer(inputConfigDict, pipelineConfigDict);
+    // TODO(CaitlinO'Callaghan): fix so player loads and test passes
+    player.configure({
+      streaming: {
+        lowLatencyMode: true,
+        inaccurateManifestTolerance: 0,
+        rebufferingGoal: 0.01,
+      }
+    });
+    await player.load(manifestUrl);
```

Comment:
You can debug in the http server in the python script that runs the tests to see what requests the player is making.  That might give us a clue as to what's wrong.

Reply:
Thank you for this insight! I will check it out.

Reply:
Ah, I figured out the issue! The `run_end_to_end_tests.py` process will sleep until the DASH or HLS stream is ready for playout. The readiness for the DASH case is determined by the function `dashStreamsReady()`. This function uses regex to check the MPD for the presence of a segment tag `<S>`, which indicates that a segment is available. LL-DASH MPDs do not contain `<S>`, so the LL-DASH stream was never able to trigger the ready state. 

To fix this, I added a check for the `availableTimeOffset` attribute in the LL-DASH case. This attribute is equal to the segment duration minus the chunk duration.  The `availableTimeOffset` value indicates to the player the earliest time that a segment can be played out, since, in the LL-DASH case, the segment can be downloaded and played before it is fully written.